app/pages/error/template.njk

```diff
@@ -3,12 +3,13 @@
 
 {% block body %}
 <div class="nhsuk-width-container nhsuk-u-margin-top-4 nhsuk-u-margin-bottom-4 bc-c-error-container nhsuk-u-padding-4">
-  <div class="nhsuk-u-padding-top-5" data-test-id="go-to-home-page-link"> 
+  <div data-test-id="error-back-link">
```

Comment:
are we going to move this to BCC at some point?

Reply:
Yeah